charts/kubernetes-agent/value-migrations/to-v2.yaml

```diff
@@ -0,0 +1,52 @@
+agent:
+  name: "{{ .agent.targetName }}"
+  deploymentTarget:
+    enabled: true
+    initial:
+      environments: [{{ quoteEach .agent.targetEnvironments  }}]
+      tags: [{{ quoteEach .agent.targetRoles }} ]
+      {{- if .agent.targetTenantedDeploymentParticipation }}
+      tenantedDeploymentParticipation: {{ .agent.targetTenantedDeploymentParticipation }}
+      {{ end}}
+      {{- if .agent.targetTenants }}
+      tenants: [{{ quoteEach .agent.targetTenants }}]
+      {{- end }}
+      {{- if .agent.targetTenantTags }}
+      tenantTags: [{{ quoteEach .agent.targetTenantTags }}]
+      {{- end }}
+      {{- if .agent.defaultNamespace }}
+      defaultNamespace: "{{ .agent.defaultNamespace}}"
+      {{- end }}  
+  targetName: null
+  targetEnvironments: null
+  targetRoles: null
+  {{- if .agent.defaultNamespace }}
+  defaultNamespace: null
+  {{- end}}
+  {{- if .agent.targetTenantTags }}
+  targetTenantTags: null
+  {{- end}}
+  {{- if .agent.targetTenants }}
+  targetTenants: null
+  {{- end}}
+  {{- if .agent.targetTenantedDeploymentParticipation }}
+  targetTenantedDeploymentParticipation: null
+  {{- end}}
+  {{- if .agent.scriptPods.image }}
+  image: null
+  {{- end}}
```

Comment:
What's this for?
Should this be `.agent.image`?

Reply:
Good spot, that's a section that I straight up should have removed.  It's actually covered by the `scriptPods:image:null` bit below.